guides/source/getting_started.md

```diff
@@ -18,7 +18,7 @@ After reading this guide, you will know:
 Guide Assumptions
 -----------------
 
-This guide is designed for beginners who want to get started with a Rails
+This guide is designed for beginners who want to get started with creating a Rails
```

Comment:
The sentence here reads just fine to me as it is.

Reply:
Please read the full line or content.
We need to make it easier to read for beginner.